.devcontainer/post_create_commands.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
-pip3 install black
-pip3 install flake8
-
 git submodule update --init --recursive
 
-python3 -m pip install --user -e .
```

Comment:
Quick note: this is used here to install Ivy as a local package, we should keep it.